packages/crud/src/vaadin-crud.js

```diff
@@ -175,8 +170,9 @@ registerStyles('vaadin-crud', crudStyles, { moduleId: 'vaadin-crud-styles' });
  * @mixes ControllerMixin
  * @mixes ElementMixin
  * @mixes ThemableMixin
+ * @mixes CrudMixin
  */
-class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement))) {
+class Crud extends ControllerMixin(ElementMixin(ThemableMixin(CrudMixin(PolymerElement)))) {
```

Comment:
nit: probably it makes sense to apply `CrudMixin` on top of other mixins (event if the order doesn't matter).

Reply:
Done